### PR TITLE
[Release] Update script to bump beta version

### PR DESCRIPTION
**Pull request by mgamis-msft**
# What
Update script to bump beta version based on the discussion in 'Web UI group' chat

# Why
We have been doing our beta version bumps incorrectly. Need this to create pre-release branch for 1.7.0-beta.1 release

# How Tested
Tested new function in playcode:
https://github.com/Azure/communication-ui-library/assets/79475487/dbb17351-20cf-4dfb-ba6f-b6f8de4d1c2c

Tested script locally:
https://github.com/Azure/communication-ui-library/assets/79475487/31c0db19-001a-4b3f-a72f-65faf27932a2

# Process & policy checklist

- [ ] I have updated the project documentation to reflect my changes if necessary.
- [ ] I have read the [CONTRIBUTING](https://github.com/Azure/communication-ui-library/blob/main/CONTRIBUTING.md) documentation.

**Is this a breaking change?**

- [ ] This change causes current functionality to break.

### CODE REVIEW

**Review by Leah-Xia-Microsoft — APPROVED** [review]
LGTM

**Review by carocao-msft — APPROVED** [review]

File: common/scripts/bump-beta-version.js [hunk 1]
```diff
@@ -13,12 +13,19 @@ const main = () => {
...
+    return [versionStrs, 0].join('-beta.');
+  }
 
-  // We will bump the patch version when create a beta on top of a stable one
-  return [[major, minor, newBeta === 0 ? newPatch : patch].join('.'), `${newBeta}`].join('-beta.');
+  const newBeta = Number.parseInt(betaVersion) + 1;
+  // We will bump the minor version when we create a beta on top of a stable one
+  const newMinor = Number.parseInt(betaVersion) === 0 ? Number.parseInt(minor) + 1 : minor;
+
+  return [[major, newMinor, 0].join('.'), `${newBeta}`].join('-beta.');
```
Comment (JamesBurnside) [inline common/scripts/bump-beta-version.js:28]: Here we are mixing string templates and string joins, to make this simpler and much easier to parse when reading, just use a string template:
```ts
return `${major}.${minor}.0-beta.${newBeta}`;
```

**Review by JamesBurnside — CHANGES_REQUESTED** [review]
Given there's a couple issue spotted, can we add some unit tests on this? Or find an npm package that can do this for us instead of having custom logic?

**Review by JamesBurnside — APPROVED** [review]
Approving to unblock creating release with the expectation we will be migrating to something more robust as at least this comment _needs_ fixed before we do our next stable release
https://github.com/Azure/communication-ui-library/pull/3280#discussion_r1243939962

**Comment by mgamis-msft** [comment]
> Approving to unblock creating release with the expectation we will be migrating to something more robust

Filed bug https://skype.visualstudio.com/SPOOL/_workitems/edit/3322843 for me to fix this script to be more robust

**Pull request merged**